### DISCUSSION TITLE
Welcome to Singlelink Discussions!

### DISCUSSION THREAD

**Discussion by jimmybisenius** [singlelink-co/Singlelink#82] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fellipeepaixao** [comment]
Hello, I'm in love with singlelink and I need help defining my custom domain, I sent you an email message, but for some reason you didn't receive it, can you help me?
